src/modules/devops-guide/pages/discovery/index.adoc

```diff
@@ -1,7 +1,6 @@
 include::ROOT:partial$variables.adoc[]
 
 = {PRODUCT_NAME} {APP_NAME_DISCOVERY}
-//Editor's note: make sure that all words in the title are capitalized (except ones like "and, for, in, a/an/the, ...")
```

Comment:
Titles, sub-titles, and all the other *-titles?

Reply:
All verbs, nouns, adjectives, adverbs and prepositions with 4 or more letters. I know it's different from how it was done before - a mistake on my part.